README.md

```diff
@@ -142,20 +142,43 @@ You must import component module you want to use inside your module to be able t
 ```diff
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
-+ import { MzButtonModule } from 'ng2-materialize';
++ import { MzButtonModule, MzInputModule } from 'ng2-materialize';
 
 import { HomeComponent } from './home.component';
 
 @NgModule({
   imports: [
     CommonModule,
 +   MzButtonModule,
++   MzInputModule,
   ],
   declarations: [ HomeComponent ],
 })
 export class HomeModule { }
 ```
 
+
+### Alternative
+As an alternative, you can import `MaterializeModule` to make all component availables. This module is **depracated** and it will be removed in a near futur.
```

Comment:
~futur~ future

Reply:
I though i did this correction haha sorry

Reply:
lol no problem ;)  This is starting to looking good ... should be all set for merge.